tests/plotting/test_plotting.py

```diff
@@ -183,7 +176,7 @@ def test_export_gltf(tmpdir, sphere, airplane):
 
 
 @skip_not_vtk9
-@skip_windows_dev_whl
+@skip_windows
 @pytest.mark.skipif(AZURE_CI_WINDOWS, reason="Windows CI testing segfaults on pbr")
 def test_pbr(sphere):
     """Test PBR rendering"""
```

Comment:
Does PBR also still segfault? (I know, wishful thinking...)

I'm wondering if we should specify these as XFAIL instead. Pro: we'd notice when they suddenly start working. Con: we'd have to open a similar PR to un-XFAIL these tests to keep CI green across all PRs at the time. (Assuming XFAIL not failing makes CI red.)

Reply:
I've not checked this, could you do that tomorrow on your VM? I'm guessing it still does.

Reason why this is a skip instead of an xfail is these tests cause an outright segfault; marking as a xfail will halt testing.

Reply:
I'm afraid the testing has to wait. I can't get `vtk` to install from pip... I might try conda, but that's more effort that I can invest right now.